crates/viewer/re_space_view_dataframe/src/expanded_rows.rs

```diff
@@ -0,0 +1,136 @@
+use std::collections::BTreeMap;
+
+/// Storage for [`ExpandedRows`], which should be persisted across frames.
+#[derive(Debug, Clone)]
+pub(crate) struct ExpandedRowsCache {
```

Comment:
Is there one of these per view, or what?

Reply:
Yes. They are stored in the view state. (Alternatively could use egui memory with view id as key, but the view state works well and doesn't require id shenanigans.)